packages/jellyfish-api-core/__tests__/category/blockchain/getTxOutSetInfo.test.ts

```diff
@@ -0,0 +1,32 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import BigNumber from 'bignumber.js'
+
+describe('TxOutSetInfo', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForBlockHeight(1)
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getTxOutSetInfo', async () => {
+    const txOutSetInfo = await client.blockchain.getTxOutSetInfo()
+
+    expect(txOutSetInfo).toStrictEqual({
+      height: expect.any(Number),
+      bestblock: expect.any(String),
+      transactions: expect.any(Number),
+      txouts: expect.any(Number),
+      bogosize: expect.any(Number),
+      hash_serialized_2: expect.any(String),
+      disk_size: expect.any(Number),
+      total_amount: expect.any(BigNumber)
```

Comment:
For values like height, I think it's better to set it to static `1` since we know it should be `1` at any time.

Reply:
You're right. Let me do a quick update.